web/src/main/java/io/fabric8/launcher/web/endpoints/OpenShiftEndpoint.java

```diff
@@ -64,7 +64,6 @@
     @GET
     @Path("/clusters")
     @Produces(MediaType.APPLICATION_JSON)
-    @Secured
```

Comment:
Removing `@Secured` will make the `authorizationInstance.get()` fail (or return null, need to test) when no `Authentication` header is specified.

Reply:
According to the spec it would throw `IllegalProductException` but due to a bug in Quarkus it throws NPE 

See https://github.com/quarkusio/quarkus/issues/3829

Reply:
not removing it will always result in a a `401`, so it will not work? what do you suggest?